src/__integrationtests__/Ctypes.spec.ts

```diff
@@ -59,7 +59,7 @@ describe('When there is an CtypeCreator and a verifier', async () => {
       } as ICType['schema'],
     } as ICType)
 
-    const iAmNotThereWowner = CType.fromCType({
+    const iAmNotThereOwner = CType.fromCType({
```

Comment:
What do you mean? 

I am not there owner or I am not the owner?

Reply:
the iAmNotThere CType but with Owner

will probably rename it to iAmNotThereWithOwner

the Wowner prolly ment to abbreviate WithOwner

Reply:
Jep!